source/common/http/http2/codec_impl.h

```diff
@@ -291,10 +291,36 @@ class ConnectionImpl : public virtual Connection,
     void encodeDataHelper(Buffer::Instance& data, bool end_stream,
                           bool skip_encoding_empty_trailers);
 
+    uint64_t sentBytes() override {
+      ENVOY_CONN_LOG(debug, "stream id {}:get sent bytes {}\n", parent_.connection_, stream_id_,
```

Comment:
we should be consistent about log levels for both codecs for this feature. H1 is `trace` while this is `debug`.